tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2663,12 +2670,14 @@ def test_whisper_longform_multi_batch_hard_prev_cond(self):
             "num_beams": 5,
         }
 
-        torch.manual_seed(0)
         result = model.generate(**inputs, **gen_kwargs)
         decoded_all = processor.batch_decode(result, skip_special_tokens=True)
 
         for i in range(num_samples):
-            assert decoded_all[i] == EXPECTED_TEXT[i]
+            if isinstance(EXPECTED_TEXT[i], str):
+                assert decoded_all[i] == EXPECTED_TEXT[i]
+            elif isinstance(EXPECTED_TEXT[i], tuple):
+                assert decoded_all[i] in EXPECTED_TEXT[i]
```

Comment:
not able to have the same results on a T4 VM and on  AWS K8S T4 runner. The difference is `I screamed` vs `I scream` so I decide to allow both of expected values.